docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 8
+title: Site-to-site installation guide for end customers
+description: Get details on why your vendor has requested you install ngrok for site-to-site connectivity, how it works, and how to maintain your end of the architecture.
+tags:
+  - guides
+---
+
+## Overview
+
+### Why am I being asked to install ngrok?
+
+Your vendor wants to create a secure persistent connection between your network and theirs, which allows them to access and take action on your services and data. We call this [**site-to-site connectivity**](https://ngrok.com/use-cases/site-to-site-connectivity).
+
+### What is ngrok?
+
+ngrok is a universal gateway—an all-in-one reverse proxy, API gateway, Kubernetes ingress, firewall, and global load balancer to deliver apps and APIs.
+
+### Why are we using ngrok instead of other solutions, like a VPN or opening ports?
+
+ngrok is simpler and more secure than these other solutions for a few reasons:
+
+- ngrok does not require you to open inbound ports in your network to the public internet.
+- ngrok’s agent is infrastructure-agnostic, operating the same way in any cloud, region, or environment.
+- The agent is simpler to configure, deploy, and maintain than VPNs or VPC peering by collapsing many networking components, like load balancing, encryption, certificate management, and authentication into a unified platform.
+- ngrok’s network includes DDoS protection and global acceleration for all connections through the [Global Server Load Balancer](https://ngrok.com/blog-post/gslb-global-server-load-balancing) (GSLB).
+
+### Who else uses ngrok to provide access to private services?
+
+Organizations worldwide trust ngrok for site-to-site connectivity, unified ingress, device gateways, and developer productivity. Our customers include Twilio, Databricks, Okta, Zoom, Microsoft, Zendesk, Cyera and many more.
+
+You can learn more about our customers and read case studies about their successes on our [customers page](https://ngrok.com/customers).
+
+### Who is ngrok? Where is it based?
+
+ngrok was first released in 2013 as an open-source project by Alan Shreve, who founded ngrok as a company in 2015 and remains our CEO. Our executive team has extensive experience at companies building distributed systems and winning organization cultures.
+
+In December 2022, ngrok [closed $50 million in Series A funding](https://techcrunch.com/2022/12/13/ngrok-a-service-to-help-devs-deploy-sites-services-and-apps-raises-50m/) with [Lightspeed Venture Partners](https://lsvp.com/) and [Coatue](https://www.coatue.com/).
+
+ngrok is headquartered in San Francisco, and a remote workforce primarily located in the U.S.
```

Comment:
ngrok is headquartered in San Francisco, and has a remote workforce primarily located in the U.S.